libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -480,8 +479,11 @@ static int load_tf_model(TFModel *tf_model, const char *model_filename)
         if (TF_GetCode(tf_model->status) != TF_OK)
         {
             TF_DeleteSession(tf_model->session, tf_model->status);
+            tf_model->session = NULL;
             TF_DeleteGraph(tf_model->graph);
+            tf_model->graph = NULL;
             TF_DeleteStatus(tf_model->status);
+            tf_model->status = NULL;
```

Comment:
Since the `tf_model->session` is deleted in here, the `tf_model->status` will not be used in `ff_dnn_free_model_tf` anymore, so free it in here is totally OK.